apps/nextjs/src/components/AppComponents/Chat/chat-right-hand-side-lesson.tsx

```diff
@@ -77,7 +77,7 @@ const ChatRightHandSideLesson = ({
           onClick={() => {
             closeMobileLessonPullOut();
           }}
-          className="flex items-center justify-center gap-3"
+          className="mt-22 flex items-center justify-center gap-3 "
```

Comment:
Should this always be 22 or does it depend on the demo status?

Reply:
Yes should be, will fix this